Tests/TuistKitAcceptanceTests/HashAcceptanceTests.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import ServiceContextModule
+import TuistAcceptanceTesting
+import XCTest
+
+final class HashAcceptanceTestXcodeProjectiOSFramework: TuistAcceptanceTestCase {
+    func test_xcode_project_ios_framework() async throws {
+        try await ServiceContext.withTestingDependencies {
+            try await setUpFixture(.custom("xcode_project_ios_framework"))
+            try await run(HashCommand.self)
+            XCTAssertStandardOutput(
+                pattern: """
+                Framework - f4c285a58fdb882b527c6d5884ef52ed
```

Comment:
I found these to be somewhat annoying to update – given how long our acceptance tests take, I'd consider checking that there's just the `Framework –` output without asserting the full hash.